lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -386,20 +389,32 @@ defmodule LivebookWeb.SessionLive.CellComponent do
     }
 
     ~L"""
-    <div class="flex items-center space-x-1">
-      <div class="flex text-xs text-gray-400">
-        <%= @text %>
-        <%= if @change_indicator do %>
-          <span data-element="change-indicator">*</span>
-        <% end %>
+    <div class="tooltip right distant" aria-label="<%= evaluated_label(opts) %>">
```

Comment:
Please add `:evaluation_time_ms` to assigns above and just pass it here as `@evaluation_time_ms`. The template should only use assigns, and no the variables directly.